dear_petition/petition/models.py

```diff
@@ -126,6 +126,8 @@ class OffenseRecord(models.Model):
     action = models.CharField(max_length=256)
     severity = models.CharField(max_length=256)
     description = models.CharField(max_length=256)
+    plea = models.CharField(max_length=256, null=True)
```

Comment:
```suggestion
    plea = models.CharField(max_length=256, blank=True)
```

Django suggests to [avoid using null on string-based fields such as CharField](https://docs.djangoproject.com/en/3.0/ref/models/fields/#null).